stories/tabs.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, Heading, Icon, Text, Tab, Tabs } from '../src';
+
+const stories = storiesOf('Tabs', module);
+
+const Box1 = (
+  <Box bg="palette.blue.lighter" p="largest">
+    Tab 1 Content
+  </Box>
+);
+const Box2 = (
+  <Box bg="palette.green.lighter" p="largest">
+    Tab 2 Content
+  </Box>
+);
+const Box3 = (
+  <Box bg="palette.red.lighter" id="foo-3" p="largest">
+    Tab 3 Content
+  </Box>
+);
+
+stories.add('Default', () => (
+  <Box as="section" p="regular">
+    <Heading.h1>Tabs</Heading.h1>
+
+    <Tabs activeTabId="tab-2">
+      <Tab content={Box1} id="tab-1">
+        Tab 1
```

Comment:
It's a bit nitpick, but I'm curious what you think of something like:

```jsx
<Tab title="Tab 1" id="tab-1">
  My tab content goes here
</Tab>
```

instead? Where title can be a node. Personally, it feels a little more clean this way. It's likely more often than not that you'd have more complex jsx / code being rendered within the Tab content itself as opposed to the title. Having to pass in all that content as a prop feels a bit off. 

Mineral-ui also has the `<Tab title` approach as well. https://mineral-ui.com/components/tabs

Reply:
I went back and forth on this but eventually landed here due to the structure of the internals of the component. When it is rendered, the content is alongside of the tab, not inside of it so the child felt a little misleading to me.

Also, I was considering implementation. I would imagine most people would put more than a bunch of text inside of the tab panel. You would probably have the content in separate files. For example, in our first use of tabs the shot details will likely be its own component, and so will the clearance information.